plugins/modules/wafv2_web_acl.py

```diff
@@ -86,6 +87,10 @@
           description:
             - Rule configuration.
           type: dict
+    custom_response_bodies:
```

Comment:
`version_added` is missing. I guess we miss the train for 2.0.0. So I guess it will be 2.1.0.